python/src/FunctionalChaosRandomVector_doc.i.in

```diff
@@ -103,65 +103,65 @@ mean= 3.50..."
 %feature("docstring") OT::FunctionalChaosRandomVector::getCovariance
```

Comment:
I cannot find the suggestions for `getMean`: in which file should I find them?

Reply:
These are the previous suggestions regarding the current file: python/src/FunctionalChaosRandomVector_doc.i.in (although many of the suggestions for getMean are now outdated because you took them into account).

Reply:
Ok: these are in eb35ae9791793d725a19037adaa2234bcb0219a5.